.travis.yml

```diff
@@ -26,6 +23,15 @@ jobs:
       env: &Debug
         - TRAVIS_CUBERITE_BUILD_TYPE=Debug
 
+    - name: "Clang 6.0 - Debug"
+      arch: arm64
+      compiler: clang
+      before_install:
+        - wget --output-document=${HOME}/CMake http://anaconda.org/conda-forge/cmake/3.17.0/download/linux-aarch64/cmake-3.17.0-h28c56e5_0.tar.bz2
```

Comment:
How about using a `CMAKE_URL` variable and use yaml references to make this `before_install` stage common to all but the osx workers?

Reply:
excellent idea

Reply:
hm, because the anaconda archive structure is different from the cmake.org one requiring a different tar flag, and because of the no-unity exception, I ended up with more lines?

Reply:
Oh well